RoboFile.php

```diff
@@ -70,19 +72,19 @@ class RoboFile extends \Robo\Tasks
 	 */
 	public function __construct()
 	{
-
 		// Set default timezone (so no warnings are generated if it is not set)
 		date_default_timezone_set('UTC');
 	}
 
 	/**
 	 * Creates a testing Joomla site for running the tests (use it before run:test)
 	 *
-	 * @param   bool $useHtaccess (1/0) Rename and enable embedded Joomla .htaccess file
+	 * @param   bool  $useHtaccess  (1/0) Rename and enable embedded Joomla .htaccess file
 	 *
+	 * @return  void
 	 * @since   3.7.3
 	 *
-	 * @return  void
+	 * @throws Exception
```

Comment:
```suggestion
	 * @throws  Exception
```
It should align. See https://developer.joomla.org/coding-standards/docblocks.html